src/drivers/cmakeDriver.ts

```diff
@@ -633,24 +640,13 @@ export abstract class CMakeDriver implements vscode.Disposable {
         await this._refreshExpansions();
     }
 
-    /**
```

Comment:
I think you should keep this comment with the variable (that moved near the constructor)